src/VecSim/info_iterator_struct.h

```diff
@@ -6,6 +6,7 @@
 
 #pragma once
 
+#include "vec_sim_common.h"
```

Comment:
if unnecessary, consider reverting (removing)

Reply:
OK. I removed from `info_iterator_struch.h`, we need it in `info_iterator.h` where we are using `uint_64_t`